### ISSUE TITLE
关于 Embedded Gaussian问题

### ISSUE THREAD

**Issue by mymuli** [AlexHex7/Non-local_pytorch#45]
想问一下，在non_local_embedded_gaussian.py实现文件中，我并没有发现Embedded **Gaussian的具体表达式，只是有矩阵乘法+softmax函数...如果是Embedded Gaussian的实验，具体是哪几行代码实现呢？
https://github.com/AlexHex7/Non-local_pytorch/blob/39ad90c91538d34e88865c9fb0ce4a844751346c/lib/non_local_embedded_gaussian.py#L86

**Comment by AlexHex7** [comment]
@mymuli  Hi, **theta_x**, **phi_x**  是embedded后的x。

**Comment by mymuli** [comment]
> @mymuli Hi, **theta_x**, **phi_x** 是embedded后的x。

我想问一下，theta_x, phi_x 是Embedded Gaussian后的x，在non_local_embedded_gaussian.py文件中具体语句是？

**Comment by AlexHex7** [comment]
在论文3.2节-Embedded Gaussian部分

> A simple extension of the Gaussian function is to compute similarity in an embedding space.

> Here θ(x_i) = W_θx_i and φ(x_j ) = W_φx_j are two embeddings

**Comment by AlexHex7** [comment]
@mymuli 在论文3.2节-Embedded Gaussian部分

> A simple extension of the Gaussian function is to compute similarity in an embedding space.

> Here θ(x_i) = W_θx_i and φ(x_j ) = W_φx_j are two embeddings

**Comment by LAB123-tech** [comment]
> 想问一下，在non_local_embedded_gaussian.py实现文件中，我并没有发现Embedded Gaussian的具体表达式，只是有矩阵乘法+softmax函数...如果是Embedded Gaussian的实现，具体是哪几行代码实现呢？
> 
> https://github.com/AlexHex7/Non-local_pytorch/blob/39ad90c91538d34e88865c9fb0ce4a844751346c/lib/non_local_embedded_gaussian.py#L85
> 
> 
> https://github.com/AlexHex7/Non-local_pytorch/blob/39ad90c91538d34e88865c9fb0ce4a844751346c/lib/non_local_embedded_gaussian.py#L86

在f_div_C = F.softmax(f, dim=-1) 之前，加个指数函数就可以了吧。
